kubernetes-manifests/frontend.yaml

```diff
@@ -88,6 +88,9 @@ spec:
           #   value: "true"
           # - name: FRONTEND_MESSAGE
           #   value: "Replace this with a message you want to display on all pages."
+          # As part of a Google Cloud demo, you can run an optional microservice called the "packaging service".
+          # - name: PACKAGING_SERVICE_URL
+          #   value: "" # This value would look like "http://123.456.789/"
```

Comment:
nit: The comment contain invalid IPv4 address. Is it intentional?

Reply:
I've updated `123.456.789` to `123.123.123`.